h2o-genmodel-extensions/xgboost/src/main/java/hex/genmodel/algos/xgboost/XGBoostMojoModel.java

```diff
@@ -48,6 +48,8 @@ public static ObjectiveType fromXGBoost(String type) {
     }
   }
 
+  public String _boosterType;
```

Comment:
IMHO these changes do warrant the MOJO version to be bumped up.

Reply:
actually the added fields are handled in a backwards compatible way, so it is really not necessary

Reply:
changing the version prevents accidentally using an older genmodel with a newer mojo

I think you are right here and there is no need to actually do that